src/sentry/utils/kafka_config.py

```diff
@@ -91,3 +91,27 @@ def get_kafka_admin_cluster_options(
     return _get_kafka_cluster_options(
         cluster_name, ADMIN_SECTION, only_bootstrap=True, override_params=override_params
     )
+
+
+def _validate_topic_definitions():
```

Comment:
```suggestion
def _validate_topic_definitions() -> None:
```

Reply:
done. we still have to do that? or shouldn't mypy infer that?

Reply:
a function with no signature is treated as untyped (`()-> Any`)